File: SuperViewer/SuperViewer/Scripts/LogEntry.ts
module LogViewerModule {
    export class LogEntry {

        constructor(public id: string,
            public env: string,
            public timestamp: Date,            
            public product: string,
            public layer: string,
            public location: string,
            public message: string,
            public hostname: string,
            public userId: string,
            public userName: string,
            public exception: string,
            public elapsedMilliseconds: number,
            public correlationId: string,
            public customException: string,
            public additionalInfo: string) {
        }
    }
}

File: SuperViewer/SuperViewer/Models/LogEntry.cs
using System;

namespace Rover.Models
{
    public class LogEntry
    {
        public string Id { get; set; }
        public DateTime Timestamp { get; set; }
        public string Product { get; set; }
        public string Layer { get; set; }
        public string Location { get; set; }
        public string Message { get; set; }
        public string Hostname { get; set; }
        public string UserId { get; set; }
        public string UserName { get; set; }
        public string Exception { get; set; }
        public int? EllapsedMilliseconds { get; set; }
        public string FormattedMessage { get; set; }
        public string CorrelationId { get; set; }        
        public string AdditionalInfo { get; set; }   
        public string Env { get; set; }
    }
}
